plugins/catalog-backend/src/service/util.ts

```diff
@@ -166,3 +171,80 @@ export function expandLegacyCompoundRelationsInEntity(entity: Entity): Entity {
   }
   return entity;
 }
+
+/**
+ * Given a list of catalog processors, filter out the ones that are disabled
+ * through the `catalog.processorOptions` config and sort them by priority.
+ */
+export function filterAndSortProcessors(
+  processors: CatalogProcessor[],
+  config: Config,
+): CatalogProcessor[] {
+  function getProcessorOptions(
+    processor: CatalogProcessor,
+  ): Config | undefined {
+    const root = config.getOptionalConfig('catalog.processorOptions');
+    try {
+      return root?.getOptionalConfig(processor.getProcessorName());
+    } catch {
+      // We silence errors specifically here, to cover for cases where the
```

Comment:
I still think it would be beneficial to log these, just for future fixes with the config loading 🤔

Reply:
I thought about it, but it kind of turns out unhelpful for the users who see those logs.

Remember, we don't log when they have already entered something in config that we can't parse - we log for exclusively internal reasons, whether the config is in place or not, just by the very act of doing `getOptional` or `has`. So we would warn about them not being able in a future hypothetical scenario to enter a certain config value. It's not actionable. And there's currently no way around it as far as I know.